config/connector_users_protobuf.config

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "datagen-protobuf-users",
+  "config": {
+    "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
+    "kafka.topic": "users",
+    "quickstart": "users",
+    "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+    "value.converter": "io.confluent.connect.protobuf.ProtobufConverter",
+    "value.converter.schemas.enable": "false",
+    "value.converter.schema.registry.url": "http://localhost:8081",
+    "producer.interceptor.classes": "io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor",
```

Comment:
FYIC remove interceptors

Reply:
I created this https://github.com/confluentinc/kafka-connect-datagen/issues/63, which I'll also track internally.  Considering there are other examples, I'll address this seperately